subprojects/internal-performance-testing/internal-performance-testing.gradle.kts

```diff
@@ -40,6 +45,8 @@ dependencies {
     compile(library("commons_httpclient"))
     compile(library("jsch"))
 
+    flamegraph("com.github.oehme:jfr-flame-graph:v0.0.5:all")
```

Comment:
What's the difference with the original flame graph tool?

Reply:
The original doesn't have a single-command build and so can't be published to jitpack. Also it can't be called twice in the same vm because it keeps static state.

Reply:
Note: This is not the flamegraph tool, but just the converter from jfr to flat stacks, which is an input for flamegraph.pl